### ISSUE TITLE
angular-google-plus improperly deals with logging out of Gmail

### ISSUE THREAD

**Issue by mgorenberg** [mrzmyr/angular-google-plus#19]
Tested in Firefox and Safari.

Procedure to replicate: 

1) Log into an account in Gmail.

2) Log into an app using angular-google-plus

3) Log out of that app.

4) Log out of Gmail

5) Attempt to log back into the app.

The app will log you in, but the window to log into a Gmail account will still appear.  If an attempt is made to log in using this window, future log-ins will fail until the browser is restarted.  Similarly, restarting the browser at any time after step 4 will resolve the issue.

**Comment by mrzmyr** [comment]
hey @mgorenberg, can you provide a plnkr.co to make your issue more clear?
Thanks

**Comment by mrzmyr** [comment]
Closing this for now, if there is still an issue with this, just create a new issue.
